norse/torch/module/conv.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import torch.nn.functional as F
+from torch.nn.common_types import _size_2_t
+from torch.nn.modules.utils import _pair
+
+
+class LConv2d(torch.nn.Conv3d):
```

Comment:
I would recommend calling it something more informed, like "LiftedConv2d" or "TimeConv2d".
Especially when there're no docs - which would also be nice ;-)

Reply:
Since this would be used a lot, I thought it would make sense to give it a short name.

Reply:
I never really understood the need for short variables. Most editors do code completion quite well. I agree we shouldn't call it ConvolutionInTwoDimensionsThatIsAlsoLiftedInTimeWhichIsCoolBecauseItIsFast :-)
Anyway, I don't have strong feelings, so if you think this is best, go ahead and GTM

Reply:
Well the idea is that you are then basically able to convert between a regular PyTorch network and a norse network by prepending a L. Basically I'm trying to make the following code nicer:
```python
model = Sequential(
    LConv2d(1, 20, 5, 1),      # Convolve from 1 -> 20 channels
    LIF(),                   # Spiking activation layer
    Lift(nn.MaxPool2d(2, 2)),
    LConv2d(20, 50, 5, 1),     # Convolve from 20 -> 50 channels
    LIF(),
    Lift(MaxPool2d(2, 2)),
    Lift( nn.Flatten()),                # Flatten to 800 units
    Lift(nn.Linear(800, 10)),
    LICell(),                    # Non-spiking integrator layer
)
```

Reply:
True, but that same logic could apply with "Lift". I do see your point. My main concern is towards newcomers who won't understand the "L". But, again, no strong feelings.

Reply:
Alternatively we could also explain that one can just use a Conv3d. But for that the shape ordering stands in the way.

Reply:
or we could omit the prefix alltogether, which would still be sort of confusing probably though

Reply:
Agreed, we should definitely distinguish it somehow. If you're happy with the code, let's just merge it as-is :)